app/main.py

```diff
@@ -26,6 +26,10 @@ def main(bot):
     APP_TOKEN = os.environ.get("APP_TOKEN")
     PREFIX = os.environ.get("PREFIX", "")
 
+    # Register Google Service command
+    bot.command(f"/{PREFIX}google-service")(google_service.google_service_command)
```

Comment:
Again, I assume this is in place just for testing, correcting?

Reply:
Exactly!